packages/contracts/as/NFT/assembly/index.ts

```diff
@@ -322,6 +327,93 @@ export function nft_revoke_all(token_id: string): void {
     }
 }
 
+/* Storage management */
+
+function internal_storage_balance_of(
+    account_id: string
+): StorageBalance | null {
+    if (accounts.get(account_id)) {
+        return <StorageBalance>{
```

Comment:
This probably needs to be saved in permanent storage as `PersistentMap<AccountId, StorageBalance>`, no?